### DISCUSSION TITLE
Method to scan image from custom file input and/or canvas

### DISCUSSION THREAD

**Discussion by ashvejnov** [mebjas/html5-qrcode#773] [General]
Hi there.

On top of generic video frame I have additional option to capture image (<input type="file" accept="image/*" capture="true">) used for different purposes (text recognition lib).

I want to let user scan barcode (if any) from photo captured to this input field. Optionally I store attached/captured photos in canvas.

How can I trigger html5qrcode like "hey, get an image from this input/canvas and try to scan barcode"?

Basically captured photo is stored in src of img as Base64 img code.

Thx.
